### PR TITLE
fix run-local after it was broke

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: deploy/deploy.sh [hunk 1]
```diff
@@ -5,8 +5,8 @@ set -x
 hco_namespace=kubevirt-hyperconverged
 
 # Create the namespaces for the HCO
-kubectl create ns $hco_namespace
-
+kubectl create ns ${hco_namespace}
+kubectl config set-context --current --namespace=${hco_namespace}
```
Comment (orenc1) [inline deploy/deploy.sh:9]: There's a reason that command was removed last time - we don't want the context to be changed to `kubevirt-hyperconverged` namespace after the deployment is done. instead, we have `-n ${hco_namespace}` on every namespaced resource below.

**Review by orenc1 — CHANGES_REQUESTED** [review]

**Review by orenc1 — APPROVED** [review]

**Comment by nunnatsa** [comment]
Changed non-testable code:
/override coverage/coveralls

**Review by orenc1 — APPROVED** [review]

**Pull request merged**